microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,627 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+This tutorial is for developers using WebView2 in immersive HoloLens 2 Unity applications.
+
```

Comment:
@michaelfarnsworth  @nishitha-burman - review the below paragraph, will be propagating it to other pages

Reply:
other spots to put this note:
* tbd

Reply:
propagated per other cmts.

done, see fresh https://github.com/MicrosoftDocs/edge-developer/pull/2567/files
you can close this Outdated comment; open fresh comment if needed